Investment-Calculator/src/components/UserInput.jsx

```diff
@@ -1,27 +1,38 @@
-export default function UserInput() {
+
+export default function UserInput({onChange, UserInput}) {
 
   return (
     <section id="user-input">
       <div className="input-group">
         <p>
           <label>Initial Investment</label>
-          <input type="number" />
+          <input type="number"
+          required
+          defaultValue={UserInput.initialInvestment}
+          onChange={(event) => onChange("inputIdentifier", event.target.value)}
+          />
         </p>
-
         <p>
           <label>Annual Investment</label>
-          <input type="number" />
+          <input type="number"
+          required
+          defaultValue={UserInput.annualInvestment}
+          onChange={(event) => onChange("inputIdentifier", event.target.value)}/>
```

Comment:
For the input field,  onChange is pre-defined and you are using it twice so for better readability update naming convention

Reply:
Will update in next PR